### ISSUE TITLE
1006k的预训练模型和334k的好像是同样一个压缩包

### ISSUE THREAD

**Issue by sunruina2** [luckycallor/InsightFace-tensorflow#26]
非常感谢代码分享，关于预训练模型里百度盘里的1006k和334k下载后的结果好像都是334k，是不是这样呢？

**Comment by sunruina2** [comment]
另外，想请教一个问题：代码中emb的最终结果是：“原图embedding归一化” + “水平翻转原图embedding归一化”，然后再归一化一次。想请教一下，其中加上水平翻转embs的作用是什么呢？
（比较担心：如果原图是45度歪头或者侧脸的矩形的话，这样会不会使得最终结果会有“四只眼睛”或重影等问题呢？但实际运行来看即使不对称的face112原图，embds_arr和embds_f_arr两个结果很相似，是因为平移不变性对么？）
embds_arr = embds_arr / np.linalg.norm(embds_arr, axis=1, keepdims=True) + embds_f_arr / np.linalg.norm(embds_f_arr, axis=1, keepdims=True)
embds_arr = embds_arr / np.linalg.norm(embds_arr, axis=1, keepdims=True)

**Comment by vvblack** [comment]
you can download the pretrained model of 1006k steps via google link
